content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -659,6 +670,45 @@ there is no automatic Job restart once the Job status is `type: Failed`.
 That is, the Job termination mechanisms activated with `.spec.activeDeadlineSeconds`
 and `.spec.backoffLimit` result in a permanent Job failure that requires manual intervention to resolve.
 
+### Terminal Job conditions
+
+A Job has two possible terminal states, it ends up either succeeded, or failed,
+and these states are reflected by the presence of the Job conditions `Complete`
+or `Failed`, respectively.
+
+The failure scenarios encompass:
+- the `.spec.backoffLimit` is exceeded
+- the `.spec.activeDeadlineSeconds` is exceeded
+- the `.spec.backoffLimitPerIndex` is exceeded (see [Backoff limit per index](#backoff-limit-per-index))
+- the Pod matches a rule in `.spec.podFailurePolicy` with the `FailJob` action (see more [Pod failure policy](#pod-failure-policy))
+
+The success scenarios encompass:
+- the `.spec.completions` is reached
+- the criteria specified by the `.spec.successPolicy` are met (see more [Success policy](#success-policy))
+
+### Termination of Job pods
+
+Prior to {{< skew currentVersion >}} the Job terminal conditions are added when the Job termination
```

Comment:
how does the `{{< skew currentVersion >}}` work? What happens when we release 1.32?

Reply:
@tengqm can you help with that?

Reply:
Ok, I tested this by looking at other places of its usage. For example this line "The kubelet in Kubernetes v1.30 collects spans from the garbage collection, pod synchronization routine as well as" in https://kubernetes.io/docs/concepts/cluster-administration/system-traces/ was added a year ago and it says "1.30", so there is no any mechanism to freeze the version.

I think our intention is to say "since v1.31", so probably it is better to hard-code it. Alternatively we can say "In Kubernetes {current version} ...". Which would you prefer @tengqm @atiratree ?

Reply:
Maybe the downside is that the documentation contains references to some very old versions like 1.21. 

OTOH, it is nice to know if some feature was introduced in 1.29 or 1.30, for example, if you are considering an upgrade.

Reply:
I don't mind hard-coding it. Currently it would give a wrong implication in 1.32, that one needs to upgrade to latest to observe the behavior.

I suppose it would be good to remove the version mention in some number of releases as the version becomes less relevant. But that it quite hard to plan.

Reply:
I agree, the wording `Prior to {{< skew currentVersion >}}` or  `Since {{< skew currentVersion >}}` will be misleading. Tentatively I use hard-coding now.